src/api/entities/__tests__/Account.ts

```diff
@@ -8,6 +8,7 @@ import { dsMockUtils, entityMockUtils } from '~/testUtils/mocks';
 import { Mocked } from '~/testUtils/types';
```

Comment:
You should test that an error is thrown when instancing if the format is not valid